mchf-eclipse/drivers/audio/audio_nr.h

```diff
@@ -24,18 +24,18 @@
 
 typedef struct NoiseReduction
```

Comment:
Hi Frank, this is not a good idea. Generally a struct is a good thing. In our case, we need two for the sake of memory allocation. 
A struct is always placed consecutively in memory, that means either ALL or NOTHING is in the special mem.  Due to the tight memory situation, we need to push as much as possible of this data in the special mem. But not all of it does fit into there. That is why not all had the __MCHF_SPECIALMEM in front of them when they were single variables.
We need to roll this back and revise your idea, otherwise we will have the memory issue again on the small machines.